addons/website_sale/models/sale_order.py

```diff
@@ -839,6 +840,47 @@ def _get_delivery_methods(self):
             *self.env['delivery.carrier']._check_company_domain(self.company_id),
         ]).filtered(lambda carrier: carrier._is_available_for_order(self))
 
+    @api.model
+    def _cron_send_order_rating_emails(self):
+        """Send rating request emails to customers a few days after order."""
+        send_rating_emails = self.env['ir.config_parameter'].sudo().get_bool('website_sale.send_rating_emails')
+        if not send_rating_emails:
+            return
+
+        days = max(0, self.env['ir.config_parameter'].sudo().get_int('website_sale.rating_email_days'))
+        date_limit = fields.Datetime.now() - timedelta(days=days)
+        rating_orders = self.search([
+            ('state', '=', 'sale'),
+            ('date_order', '>=', date_limit.replace(hour=0, minute=0, second=0)),
+            ('date_order', '<=', date_limit.replace(hour=23, minute=59, second=59)),
+            ('website_id', '!=', False),
+            ('rating_email_sent', '=', False),
+        ])
+
+        for order in rating_orders:
+            mail_template = order._get_rating_request_template()
+            if mail_template:
+                mail_template.send_mail(order.id)
+                self.rating_email_sent = True
+
+    def is_view_active_for_website(self, view_key):
+        self.ensure_one()
+        website = self.website_id or self.env['website'].get_current_website()
+        views = self.env['ir.ui.view'].search([('key', '=', view_key)])
+        return any(v.active and (not v.website_id or v.website_id == website) for v in views)
+
+    def _get_rating_request_template(self):
+        self.ensure_one()
+
+        default_rating_template_id = self.env['ir.config_parameter'].sudo().get_int(
```

Comment:
Nit (this isn't necessarily the default one):
```suggestion
        rating_email_template_id = self.env['ir.config_parameter'].sudo().get_int(
```